### PR TITLE
[v6r15] Circumvent globus bug in gfal2

### PR DESCRIPTION
**Pull request by chaen**
* rename self.gfal2 into self.ctx 
* There is a bug in globus. I added a magic line that circumvent it until fixed (if you want the full story: https://its.cern.ch/jira/browse/DMC-853)

### CODE REVIEW

File: Resources/Storage/GFAL2_StorageBase.py [hunk 1]
```diff
@@ -62,10 +62,14 @@ def __init__( self, storageName, parameters ):
...
 
     # # gfal2 API
-    self.gfal2 = gfal2.creat_context()
+    self.ctx = gfal2.creat_context()
 
     # by default turn off BDII checks
-    self.gfal2.set_opt_boolean( "BDII", "ENABLE", False )
+    self.ctx.set_opt_boolean( "BDII", "ENABLE", False )
+
+    # Avoid caching because of a bug in globus (https://its.cern.ch/jira/browse/DMC-853)
```
Comment (fstagni) [inline Resources/Storage/GFAL2_StorageBase.py]: Maybe add in the comment a "FIXME:" so maybe one day we'll be reminded to remove this line.

**Comment by fstagni** [comment]
Out of curiosity, why renaming it ctx?

File: Resources/Storage/GFAL2_StorageBase.py (hunk 1 above)
Comment (chaen) [inline Resources/Storage/GFAL2_StorageBase.py]: ok

**Comment by chaen** [comment]
Because I think it makes it more readable. You know that you are working with the context, and not the gfal2 library itself. In fact, in the original SRM plugin, you had ```self.lcg_util``` and ```self.gfal``` that were the libraries, and I found that really ugly, so I wanted to clearly distinguish here. It was something like bellow
```
def__init__(self): 
  import lcg_util
  self.lcg_util = lcg_util
```

**Comment by fstagni** [comment]
Review OK for the pure code (but notice that the actual content is, to my eyes black magic).

**Pull request merged**